### DISCUSSION TITLE
Use of authorization

### DISCUSSION THREAD

**Discussion by jdkorigan** [Azure/data-api-builder#1435] [Q&A]
Hello,
I want to use authorization in dab-config file, but it is not clear to me how to use it. 
Would you have a simple example to show how to proceed with AD? (for docker)

1. the first thing to do, I think is to add the following lines in config file :

"runtime": {
    "host": {
      "authentication": {
        "provider": "AzureAD",
        "jwt": {
          "audience": "AudienceGUIDfromAppRegistration",
          "issuer": "https://login.microsoftonline.com/tenantID/v2.0"
        }
      }
    }

**what is "AudienceGUIDfromAppRegistration" where can we get it in Azure?**

Best Regards,
Jérôme

**Comment by seantleonard** [comment]
Hi @jdkorigan,

When you say AD, do you mean Azure AD? If so, you would have to navigate to the App Registration you created for your instance of Data Api builder. On your app registration's Overview page, you would use the GUID listed in the **Application (client) ID** field. 
![image](https://user-images.githubusercontent.com/6414189/233253178-3e843ca2-e4f3-4eb0-80b8-5c20b27bf9fd.png)

The documentation for configuring Azure AD app registrations with Data Api builder can be found here: https://github.com/Azure/data-api-builder/blob/main/docs/authentication-azure-ad.md. If you've already tried to reference that, let me know what you think can be improved.
